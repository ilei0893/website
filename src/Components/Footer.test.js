import React from "react";
import Footer from "./Footer.js";
import renderer from "react-test-renderer";

describe("Tests the footer section", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
