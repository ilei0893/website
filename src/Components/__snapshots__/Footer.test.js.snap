// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests the header section matches the snapshot 1`] = `
<footer>
  <div
    id="go-top"
  >
    <a
      className="smoothscroll"
      href="#home"
      title="Back to Top"
    >
      <i
        className="icon-up-open"
      />
    </a>
  </div>
</footer>
`;
